spec/api-browser-window-spec.js

```diff
@@ -3276,29 +3277,29 @@ describe('BrowserWindow module', () => {
     })
 
     it('separates the page context from the Electron/preload context', async () => {
-      iw.loadURL(`file://${fixtures}/api/isolated.html`)
+      iw.loadFile(path.join(fixtures, 'api', 'isolated.html'))
       const [, data] = await emittedOnce(ipcMain, 'isolated-world')
       assert.deepEqual(data, expectedContextData)
     })
     it('recreates the contexts on reload', async () => {
-      iw.loadURL(`file://${fixtures}/api/isolated.html`)
+      iw.loadFile(path.join(fixtures, 'api', 'isolated.html'))
       await emittedOnce(iw.webContents, 'did-finish-load')
       iw.webContents.reload()
       const [, data] = await emittedOnce(ipcMain, 'isolated-world')
       assert.deepEqual(data, expectedContextData)
     })
     it('enables context isolation on child windows', async () => {
-      iw.loadURL(`file://${fixtures}/pages/window-open.html`)
+      iw.loadFile(path.join(fixtures, 'pages', 'window-open.html'))
       const [, window] = await emittedOnce(app, 'browser-window-created')
       assert.ok(window.webContents.getLastWebPreferences().contextIsolation)
     })
     it('separates the page context from the Electron/preload context with sandbox on', async () => {
-      ws.loadURL(`file://${fixtures}/api/isolated.html`)
+      iw.loadFile(path.join(fixtures, 'api', 'isolated.html'))
```

Comment:
You used a different windows here: `iw` instead of `ws`.
And in a couple of other cases too )

Reply:
fixed